students/15th/team3/user/views.py

```diff
@@ -0,0 +1,46 @@
+from django.shortcuts import render
+import re
+import json
+from django.http     import JsonResponse
+from .models   import SignUp
+from django.views    import View
```

Comment:
- `import` 순서도 가독성에 영향을 미칩니다. 아래처럼 수정해주세요.
```suggestion
import re, json # 파이썬 자체 모듈

from django.http  import JsonResponse # 장고 제공 모듈
from django.views import View

from .models import SignUp # 직접 구현한 클래스
```

Reply:
넵넵